### ISSUE TITLE
ZulipRemoteUserBackend is currently not supported

### ISSUE THREAD

**Issue by ssvenn** [zulip/docker-zulip#210]
I am scratching my head trying to figure out how to implement the Apache SSO with docker-zulip, would it be better to bundle apache and libapache2-mod-auth-kerb inside the container or try to run it in a separate container or on the docker host?

https://zulip.readthedocs.io/en/latest/production/authentication-methods.html#apache-based-sso-with-remote-user

I am going to attempt modifying the app server container to include the required apache components but my understanding of kerberos is limited so I'm unsure if that's the best strategy.
I have LDAP working but I really want to get SSO working so I can package the desktop client to autostart and autologin on our Windows machines.

**Comment by timabbott** [comment]
Yeah, I think you'd need to fork the Dockerfile and add code to include your Apache stuff to do this.

(Or write a new Dockerfile extending the main Zulip image, which might be cleaner)
